Makefile

```diff
@@ -79,16 +79,6 @@ crds.clean:
 
 generate.run: go.generate crds.clean
 
```

Comment:
```suggestion
generate.run: services.all go.generate crds.clean

```

We intentionally override `reviewable` here to make sure generated code is up to date. Maybe add `services.all` to `generate.init` or `generate.run`?

Reply:
Ah right - will fix.

Reply:
I took a stab at this, but it's not quite working how I'd like. For whatever reason `go generate` runs _before_ `services.all` - rather than after possibly because it's part of the Go make libraries?

Reply:
Found a way to fix this.